src/main/js/main_html5.js

```diff
@@ -150,7 +150,7 @@ require("../../../html5-common/js/utils/environment.js");
     var firstPlay = true;
     var playerDimension = dimension;
     var videoDimension = {height: 0, width: 0};
-    var queuedInitialTime = 0;
+    var initialTime = { value: 0, reached: true };
```

Comment:
Was there a bug that warranted this change or it is mainly a cleanup? (I am seeing this this is a cleaner approach)